### ISSUE TITLE
Filter/error on invalid types in wizard

### ISSUE THREAD

**Issue by rugk** [tyrasd/overpass-turbo#403]
When I used a wizard query with `type:ways` instead of the correct `type:way` then overpass-turbo did not show me an error, but just generated an "empty" query.

## What happens
Just a bounding box with nothing (except of comments) in it, has been generated.

## What should happen
Show an error or so that the syntax is invalid. Probably hinting to the wrong `type`.

Reproduced on http://overpass-turbo.eu/

**Comment by mnalis** [comment]
Absolutely... I've just had an issue with: `route = ferry and foot is null and (type:way or type:route)` which silently ingored one of the types... (of course, it should've been `type:relation`, but lack of error checking is likely to lead to either unnoticed incorrect results, or in chasing ghosts - both of which are highly undesirable).

In general I think that if wizard is **unable to parse any string, it should error out, instead of _silently_ ignoring it**.
